converter/fluka/card.py

```diff
@@ -0,0 +1,28 @@
+from dataclasses import dataclass
+from typing import Optional, List
+
+
+@dataclass
+class Card:
+    """Class for a fluka card."""
+
+    tag: str = ""
+    what: Optional[List[str]] = None
```

Comment:
Why don't you change type to `List[str]` and put a default value of `[]` ?

Reply:
I was receiving an error "mutable default for field is not allowed" so I changed to that. I must admit that at first I didn't see full error message which provided a solution, which I implemented now  `what: list[str] = field(default_factory=list)`